<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-04-13T12:08:58 IST" name="Suite" finished-at="2023-04-13T12:09:50 IST" duration-ms="51970">
    <groups>
    </groups>
    <test started-at="2023-04-13T12:08:58 IST" name="Test" finished-at="2023-04-13T12:09:50 IST" duration-ms="51970">
      <class name="insuranceCalculation_Motorcucle.ErrorTests_motorcycle">
        <test-method is-config="true" signature="setupMethod()[pri:0, instance:insuranceCalculation_Motorcucle.ErrorTests_motorcycle@3b0fe47a]" started-at="2023-04-13T12:08:58 IST" name="setupMethod" finished-at="2023-04-13T12:09:11 IST" duration-ms="12446" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupMethod -->
        <test-method signature="errorMessage_PayLoadField()[pri:0, instance:insuranceCalculation_Motorcucle.ErrorTests_motorcycle@3b0fe47a]" started-at="2023-04-13T12:09:11 IST" name="errorMessage_PayLoadField" finished-at="2023-04-13T12:09:13 IST" duration-ms="2123" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- errorMessage_PayLoadField -->
        <test-method is-config="true" signature="cleanUP()[pri:0, instance:insuranceCalculation_Motorcucle.ErrorTests_motorcycle@3b0fe47a]" started-at="2023-04-13T12:09:13 IST" name="cleanUP" finished-at="2023-04-13T12:09:14 IST" duration-ms="695" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanUP -->
        <test-method is-config="true" signature="setupMethod()[pri:0, instance:insuranceCalculation_Motorcucle.ErrorTests_motorcycle@3b0fe47a]" started-at="2023-04-13T12:09:14 IST" name="setupMethod" finished-at="2023-04-13T12:09:45 IST" duration-ms="31416" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupMethod -->
        <test-method signature="errorMessage_PayLoadField1()[pri:0, instance:insuranceCalculation_Motorcucle.ErrorTests_motorcycle@3b0fe47a]" started-at="2023-04-13T12:09:45 IST" name="errorMessage_PayLoadField1" finished-at="2023-04-13T12:09:49 IST" duration-ms="4130" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Error Message is not expected on cylinder capacity field expected [Must be a number between 1 and 3000] but found [Must be a number between 1 and 2000]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Error Message is not expected on cylinder capacity field expected [Must be a number between 1 and 3000] but found [Must be a number between 1 and 2000]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at insuranceCalculation_Motorcucle.ErrorTests_motorcycle.errorMessage_PayLoadField1(ErrorTests_motorcycle.java:36)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- errorMessage_PayLoadField1 -->
        <test-method is-config="true" signature="cleanUP()[pri:0, instance:insuranceCalculation_Motorcucle.ErrorTests_motorcycle@3b0fe47a]" started-at="2023-04-13T12:09:49 IST" name="cleanUP" finished-at="2023-04-13T12:09:50 IST" duration-ms="717" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanUP -->
      </class> <!-- insuranceCalculation_Motorcucle.ErrorTests_motorcycle -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
